src/Microsoft.CodeQuality.Analyzers/UnitTests/Maintainability/AvoidUnusedPrivateFieldsTests.cs

```diff
@@ -416,13 +439,16 @@ <System.ComponentModel.Composition.ExportAttribute> _
     Private fieldWithMefV2ExportAttribute As Integer
 End Class
 ",
-                CompilerDiagnostics = CompilerDiagnostics.None,
                 ExpectedDiagnostics =
                 {
+                    // Test0.vb(3) : error BC30002: Type 'System.Composition.ExportAttribute' is not defined.
+                    DiagnosticResult.CompilerError("BC30002").WithLocation(3, 6),
                     // Test0.vb(4,13): warning CA1823: Unused field 'fieldWithMefV1ExportAttribute'.
                     GetCA1823BasicResultAt(4, 13, "fieldWithMefV1ExportAttribute"),
+                    // Test0.vb(6) : error BC30002: Type 'System.ComponentModel.Composition.ExportAttribute' is not defined.
+                    DiagnosticResult.CompilerError("BC30002").WithLocation(6, 6),
                     // Test0.vb(7,13): warning CA1823: Unused field 'fieldWithMefV2ExportAttribute'.
-                    GetCA1823BasicResultAt(7, 13, "fieldWithMefV2ExportAttribute"),
+                    GetCA1823BasicResultAt(7, 13, "fieldWithMefV2ExportAttribute")
```

Comment:
📝 Trailing commas on the last item of a multi-line collection initializer should not be removed.

```suggestion
                    GetCA1823BasicResultAt(7, 13, "fieldWithMefV2ExportAttribute"),
```